Karma/BusinessLogic/DB/User.cs

```diff
@@ -12,10 +12,17 @@ namespace BusinessLogic
     /// </summary>
     public class User
     {
-        string Name { get; set; }
-        string LastName { get; set; }
-        string UserName { get; set; }
-        string password { get; set; }
+        public User (string _Name, string _LastName, string _UserName, string _Password)
+        {
+            Name = _Name;
+            LastName = _LastName;
+            UserName = _UserName;
+            Password = _Password;
+        }
+        public string Name { get; set; }
```

Comment:
In my opinion, these fields should be at the top, and then the constructor. So the order would go like this
Fields -> Constructor -> various methods